src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss

```diff
@@ -5,6 +5,10 @@
   cursor: pointer;
   font-size: rem(12);
   align-items: center;
+  .user-avatar{
```

Comment:
this class should come for parent container with width and height. For img we need to give width:100% otherwise image will be cut